client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -43,6 +43,7 @@ function addQueryByAPI(data, shouldPublish = true) {
     schedule: null,
   }, data);
 
+  // eslint-disable-next-line cypress/no-assigning-return-values
```

Comment:
`cypress/recommended` seemed nice :confused:. As we can't use it, should those cypress `eslint-disable` be removed?

Reply:
Maybe it's cause the cypress package isn't a required installation.

Reply:
Might be 🤔